Gemfile

```diff
@@ -1,5 +1,11 @@
 source 'https://rubygems.org'
 
+gem 'pry'
```

Comment:
Not sure we should make `pry` a dependency - you don't need it to run specs, and it'd be nice to keep dependencies to a minimum. Not hard to just install pry yourself if you want to use it when debugging.

Reply:
Eh. I like having it in all my gems when debugging them, and it adds no runtime dependency. If you feel strongly, I'll remove it.

Reply:
Nah, I don't feel super strongly, just a (possibly mostly aesthetic) objection that it's not an _actual_ dependency, just a nice to have that people could add themselves if they wanted to, but NABD.